index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portfolio Page</title>
     <link rel="stylesheet" href="index.css">
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
```

Comment:
- [Optional] Always use an indentation of two spaces in all your documents. This is a convention and also [best practice]([url](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)).

Reply:
Noted! thanks for the review @Kamba56 !